tests/integration/logging.tcl

```diff
@@ -1,14 +1,26 @@
-set server_path [tmpdir server.log]
 set system_name [string tolower [exec uname -s]]
 # ldd --version returns 1 under musl for unknown reasons. If this check stops working, that may be why
 set is_musl [catch {exec ldd --version}]
+set system_supported 0
 
-if {$system_name eq {linux} && $is_musl eq 0 || $system_name eq {darwin}} {
+# We only support darwin or Linux with glibc
+if {$system_name eq {darwin}} {
+    set system_supported 1
+} elseif {$system_name eq {linux} && $is_musl eq 0} {
```

Comment:
The is_musl variable should be removed here since redis is now just detecting it with the ldd command below. Seems safer that way since ldd --version relies on a hacky workaround anyway.